doc/whatsnew.rst

```diff
@@ -15,6 +15,11 @@ Next release
   - Add missing basins_country_R12.csv data file to :mod:`.model.water.data` (:pull:`371`).
   - Convert Excel data files to CSV format and add improve test coverage for :mod:`.model.water.data` (:pull:`371`).
 
+- Improve and extend :doc:`/material/index`.
+  See :doc:`/model/material/v1.2.0` for details.
```

Comment:
Per the preview of the built docs, this link is broken. I'll fix it while also fixing https://github.com/iiasa/message-ix-models/pull/371/files#r2251010279